frontend/src/app/store/reducers/outcomes/assessment.ts

```diff
@@ -0,0 +1,67 @@
+import { Assessment } from "../../../types/outcomes/assessment";
+import { AssessmentState } from "../../../types/state/assessment-state";
+import { AppThunk, RootState } from "../../store";
+import { createAction, createSlice } from "@reduxjs/toolkit";
+import config from "../../../../config";
+import { generateApiParameters, get } from "../../../common/api";
+
+const initialState: AssessmentState = {
+  assessment: {
+    action_required: undefined,
+    date: undefined,
+    justification: undefined,
+    officer: undefined,
+    assessment_type: []
+  }
+};
+
+export const assessmentSlice = createSlice({
+  name: "assessment",
+  initialState,
+  // The `reducers` field lets us define reducers and generate associated actions
+  reducers: {
+    setAssessment: (state, action) => {
+      const { payload } = action;
```

Comment:
Here you just want to spread your assessment instead of creating an assessment object so you'd do somethign like

const { payload: { assessment } } = action
return { ...assessment }